docs/data-modeling.md

```diff
@@ -0,0 +1,66 @@
+# Model your data
+Data modeling (i.e., design your database schemas) is the process of conceptualizing and visualizing how data will be stored and used by identifying the patterns used to access data and the types of queries to be performed within business operations. 
+
+This page first explains the ScalarDB data model and then describes how to design your database schemas based on the data model.
+
+## ScalarDB data model
+ScalarDB’s data model is an extended key-value model inspired by the Bigtable data model. It is similar to the relational model but differs in several ways, as described below.
+
+The following diagram shows an example of ScalarDB tables, each a collection of records. This section first explains what objects, such as tables and records, ScalarDB defines and then describes how to locate records.
+
+![ScalarDB data model.](images/scalardb_data_model.png)
+
+### Objects in ScalarDB
+The ScalarDB data model has several objects.
+
+#### Namespace
+A namespace is a collection of tables analogous to an SQL namespace or database. Typically, each application creates all its tables in one namespace.
```

Comment:
I think another characteristic of ScalarDB's Namespace is that an arbitrary underlying database/storage can be used for each Namespace by leveraging the multi storage feature. I'm not sure the topic is out of the scope of this document, though.

Reply:
I agree with @komamitsu .
My impression of Namespace is that it helps abstract multiple databases as if they were a single database.

Reply:
Thanks! Namespace mapping things should probably be described in the multi-storage transaction docs to make this doc more focused.
But, since ScalarDB tends to manage multiple namespaces, mapped from different databases, I will remove `Typically, each application creates all its tables in one namespace`.